Add Prasanth maintainer of Harbor

Vad1mo opened the pull request:
Based on the [LFX Project Insights,](https://insights.linuxfoundation.org/project/harbor/repository/goharbor-harbor/contributors?timeRange=past180days&start=2024-12-27&end=2025-06-25) Prashanth was a top 1, top 2 contributor to Harbor's recent advancements. 

This PR's is adding Prashanth to recognize his contribution to CNCF Harbor.

- Project Harbor
- Harbor CLI 


![image](https://github.com/user-attachments/assets/ac5ec406-970e-419e-bbd7-780e494e434c)
![image](https://github.com/user-attachments/assets/0cea5101-ee97-456d-9dae-fa8de1dae740)

Vad1mo committed:
add Prasanth maintainer of Harbor

Signed-off-by: Vadim Bauer <vb@container-registry.com>

OrlinVasilev reviewed (approved):
Well deserved!!!

thcdrt reviewed (approved):
👍

Vad1mo commented:
> I have reached out to Vadim and Prasanth via Email and PR about this proposal.

Thank you for sharing your view with me. Please publish your statement on the PR so the feedback reaches Prasanth too, so he has an idea of what he needs to improve/complete in order to be accepted as a maintainer.

Vad1mo committed:
Add proposal for image patching feature using Copacetic

Signed-off-by: Vadim Bauer <vb@container-registry.com>

reasonerjt commented:
> Thank you for sharing your view with me. Please publish your statement on the PR so the feedback reaches Prasanth too, so he has an idea of what he needs to improve/complete in order to be accepted as a maintainer.

I've already sent the messages to Prasanth via Slack DM.  I did it in early Aug.

reasonerjt reviewed (dismissed):
Putting this on hold per communication with Vadim and Prasanth.

Vad1mo committed:
Remove Copacetic image patching proposal from maintainer PR

This PR is scoped to adding Prasanth as a Harbor maintainer. The
proposal was committed to this branch by mistake.

Signed-off-by: Vadim Bauer <vb@container-registry.com>

Vad1mo committed:
Merge branch 'main' into new_maintainer_prasanth

wy65701436 reviewed (approved):
lgtm

chlins reviewed (approved):
lgtm
